README.md

```diff
@@ -3,6 +3,19 @@ Criteria for selecting CI/CD tools
 
 Here's a collection of thoughts regarding how to chose a CI/CD tool that fits your needs.
 
+Supported Concepts
+------------------
+
+* pipline per commit
+* Stages
+* Jobs
+* Artifact storage and retrival for a job (S3, on CICD Server, on artifact server ...), support for artifact lifecycle
+* Caching concept
+* Support for deployments (redeployment of older commits x stage)
+* Quality of statemachine concepts
```

Comment:
can you maybe add a few words to this - I am not quite sure, what this is about :)